examples/4337-passkeys/src/hooks/useCodeAtAddress.ts

```diff
@@ -41,7 +41,10 @@ function useCodeAtAddress(provider: ethers.Eip1193Provider, address: string, opt
       const pollInterval = opts?.pollInterval || 5000
 
       updateBalance()
-      const interval = setInterval(updateBalance, pollInterval)
+      // We convert the interval to a number because typescript thinks this call is ambiguous
+      // since node.js and browser have different return types for setInterval
+      // more info: https://github.com/Microsoft/TypeScript/issues/30128#issuecomment-807394387
+      const interval = +setInterval(updateBalance, pollInterval)
```

Comment:
I wish this piece of "knowledge" only appeared once (like in a wrapper function or something). This is an example though, so not super important and maybe not worth the overhead.